torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6890,6 +6890,23 @@ def gen_shape_kwargs():
     for input, target, var, kwargs in gen_shape_kwargs():
         yield SampleInput(input, args=(target, var, ), kwargs=kwargs)
 
+def error_inputs_gaussian_nll_loss(op_info, device, **kwargs):
+    _make = partial(make_tensor, device=device, dtype=torch.float32)
```

Comment:
style nit. I think most opinfos call this `make_arg`, but not necessary to change it really.